rpc/src/rpc.rs

```diff
@@ -1761,8 +1761,8 @@ impl JsonRpcRequestProcessor {
                             bigtable_before = None;
                         }
                         Err(err) => {
-                            warn!("{:?}", err);
-                            return Ok(map_results(results));
+                            warn!("Failed to query Bigtable: {:?}", err);
+                            return Err(RpcCustomError::LongTermStorageUnreachable.into());
```

Comment:
Now that `storage_bigtable/lib.rs` properly throws `SignatureNotFound` errors, we can be sure that when this arm is matched, it's because of a Bigtable connection error.